src/athena.hpp

```diff
@@ -78,6 +78,26 @@ struct LogicalLocation { // aggregate and POD type
 //! prototype for overloading the comparison operator (defined in meshblock_tree.cpp)
 bool operator==(const LogicalLocation &l1, const LogicalLocation &l2);
 
+
```

Comment:
Could you say a bit about why this moved from being a (mostly?) "multigrid-specific" feature?
I see that the `LogicalLocationHash` and `unordered_map`'s  are currently used in Multigrid and `meshblock_tree.*` Can you say more about the maps / can the same maps be shared across these classes?

Reply:
@tomidakn did you see this question?

Reply:
Sorry for my late response. I moved this as I just feel this operator should be associated with the LogicalLocation structure like other operators. The map is used to quickly find a neighboring MeshBlock without searching the whole Tree.